src/sentry/tagstore/snuba/backend.py

```diff
@@ -175,15 +175,12 @@ def __get_tag_keys_for_projects(self, projects, group_id,
         if keys is not None:
             filters['tags_key'] = keys
         aggregations = [
-            ['uniq', 'tags_value', 'values_seen'],
             ['count()', '', 'count']
         ]
         conditions = [['tags_key', 'NOT IN', self.EXCLUDE_TAG_KEYS]]
 
-        # TODO should this be sorted by count() descending, rather than the
-        # number of unique values
         result = snuba.query(start, end, ['tags_key'], conditions, filters,
-                             aggregations, limit=limit, orderby='-values_seen',
+                             aggregations, limit=limit, orderby='-count',
```

Comment:
These changes look good, but this method is also consumed by `__get_tag_keys`, which is used in a bunch of places, including group tags which I imagine actually cares about these aggregate values. These changes basically need to be confined to `get_tag_keys_for_projects`. We could just parameterize this method, something like `include_tag_counts`?